.storybook/main.js

```diff
@@ -5,45 +5,3 @@ module.exports = {
     "storybook-addon-styled-component-theme/dist/register",
   ],
 };
-
-// import { configure, setAddon, addDecorator } from "@storybook/react";
```

Comment:
This commit can be fixup with the first one

Reply:
I should have fixup some cleaning commits indeed sorry.